Assets/Scripts/Modules/Combat/ICombatModule.cs

```diff
@@ -1,8 +1,19 @@
 ﻿/*
  * Authors: Ian Macpherson, Isaiah Mann
- * Description: An interface to handle the turn logic
+ * Description: An interface to handle the combat logic
  */
 
 public interface ICombatModule {
```

Comment:
All methods should be modified so this model can be single instance. Feel free to also submit a separate interface that will be used by an individual unit (same deal for StatModule)